src/resolvers-types.ts

```diff
@@ -1663,10 +1662,21 @@ export type Location = {
   servicePoints?: Maybe<Array<ServicePoint>>;
 };
 
+export enum LocationAvailabilityClass {
+  Available = 'Available',
+  InProcess = 'In_Process',
```

Comment:
Is there a reason this is the way it is?  Aesthetically, I prefer any other alternative:
```
In Process
In-Process
InProcess
in_process
```

Reply:
All the other enums are auto-generated from the FOLIO JSON APIs and generally follow this aesthetic.